### DISCUSSION TITLE
Error | Exception: Unable to talk to trigger service

### DISCUSSION THREAD

**Discussion by jBbL81** [Chimildic/goofy#260] [Помощь с ошибкой]
![Screenshot 2024-07-16 164948](https://github.com/user-attachments/assets/8817954d-7d0b-4fbc-b9a4-1fb9f28a8e75)

**Comment by Chimildic** [comment]
The error links to `ScriptApp` module that build in to GAS. So goofy works as before. Just wait when Google fix service issue.
